### DISCUSSION TITLE
LTE-Protection auf FritzBox Cable

### DISCUSSION THREAD

**Discussion by qdlmcfresh** [Freetz-NG/freetz-ng#503] [Allgemein] [answered]
Ich habe an meinem Vodafone Kabel anschluss wohl LTE-Einstrahlung und nach dem Besuch eines Technikers, für den ich den Arris router verbinden musste, wurde mir auf diesem wohl eine LTE-Protection eingestellt um die gestörten kanäle zu ?blockieren? Könnte man sowas auch auf der FritzBox implementieren oder ist ein LTE-Blocker direkt am Kabel die einzige möglichkeit?

**Comment by fda77** [comment]
Einfach im Aldi ne dicke Rolle Alufolie kaufen und damit die Fritzbox ordentlich einwickeln. Darf aber nicht überhitzen, also kühl lagern!
Danach kommt kein LTE mehr rein - und kein Wlan + Dect mehr raus.
Meist strahlt es aber irgendwo ins Kabel ein und den providern ist es zu teuer dies zu suchen und reparieren

**Reply by qdlmcfresh** [reply]
Hab die box jetzt im Kühlschrank. Alle störsignale sind weg, danke

**Comment by fda77** [comment]
Dir bleibt nichts anderes übrig als deinen Provider zu nerven bis er sich dem Problem annimmt. Falls es wirklich direkt in die Fritzbox strahlen sollte, ist sie defekt. Dafür ist AVM zuständig

**Reply by qdlmcfresh** [reply]
![grafik](https://user-images.githubusercontent.com/10837173/160430894-799b9fa8-441d-4b97-b30e-94b2e4771bad.png)

Vlt die frage nochmal anders formuliert. Gibt es eine softwareseitige möglichkeit der fritzbox zu verbieten die gestörten kanäle zu verwenden?

**Comment by fda77** [comment, marked as answer]
Ich kenne keinen, aber vielleicht fragst du besser in einem Cable-Forum. Hast du das mit der Alufolie oder dem Kühlschrank mal ausprobiert? Ist die Schirmung in der Fritzbox Müll oder das Kabel irgendwo "um die Ecke geknickt"? Von Biegeradius haben nur weniger jemals was gehört.
Andere Herangehensweise: sperr die Frequenzen vor der Fritzbox. zb mit einer alten Multimediadose oder Verstärker der nur bis 450 oder 600 durchlässt :D Oder besser einem extra Sperrfilter wie zb https://www.amazon.de/dp/B00EDER4QM das funktioniert mit jedem Modem

**Reply by qdlmcfresh** [reply]
Ne das mit der Alufolie/Kühlschrank hab ich nicht getestet. Denke aber auch nicht, dass die einstrahlung von der FritzBox kommt, da ich die auch schon gewechselt habe und im Vodafone Router das problem auch besteht. (auch mit anderen Kabeln und direkt an der Buchse im Keller) Deshalb haben die ja, wenn ich das mit der LTE-Protection richtig verstehe, die gestörten Kanäle softwareseitig deaktiviert. Den Sperrfilter  hatte ich auch schon im Auge aber fand an der Lösung eben unschön, dass nicht nur die 794 & 802 MHz raußfliegen sondern alles darüber auch... Vlt macht das aber auch keinen riesen unterschied ...

**Reply by fda77** [reply]
Hm, das verlinkte Teil sperrt von 700-790, das passt bei dir nicht so gut! Vielleicht gibt es da noch andere mit höherem Bereich. 800Mhz wird wohl ehe 5G sein.
Jedenfalls können in ein Coaxkabel keine Störungen von DVB/LTE/5G reinkommen solang die (aktuell 3 oder 4 fache) Schirmung okay ist! Ich würde den Provder weiter nerven, das ist seine Sache

**Reply by fda77** [reply]
Übrigens sind USB-Kabel an der Fritzbox gute Antennen! Deshalb funktioniert USB3 + 5GHz Wlan _gleichzeitig_ nicht so gut

**Reply by qdlmcfresh** [reply]
> Übrigens sind USB-Kabel an der Fritzbox gute Antennen! Deshalb funktioniert USB3 + 5GHz Wlan _gleichzeitig_ nicht so gut

Ah guter Hinweis, das kanns leider auch nicht sein :D Da steckt nix ...



> Hm, das verlinkte Teil sperrt von 700-790, das passt bei dir nicht so gut! Vielleicht gibt es da noch andere mit höherem Bereich. 800Mhz wird wohl ehe 5G sein. Jedenfalls können in ein Coaxkabel keine Störungen von DVB/LTE/5G reinkommen solang die (aktuell 3 oder 4 fache) Schirmung okay ist! Ich würde den Provder weiter nerven, das ist seine Sache

Bin gerade dabei den Vodafone Router auf den Bridge modus umzustellen, so kann ich meine fritzbox weiterverwenden und bin supportberechtigt. Solange die Fritzbox direkt am anschluss steckt wollen die nichts von mir hören weil Eigengerät

**Comment by fda77** [comment]
Da musst du echt hartnäckig sein. Vodafone hat sich seinen Supportunwillen von der Telekom abgeschaut.
Ich hatte mit Vodafon auch schon Monate lang Ärger weil deren DHCP nur noch IPv4 rausgab.
Am wirksamsten ist übrigens wenn weniger Geld überweist (Minderung)! Natürlich erst nach einer angemessenen Frist von zb 2 Wochen zur Fehlerbehebung und Nachweis, am besten mit der Breitband "Desktop-App".

Die Männchen in den rosa Hemdchen sind übrigens der Meinung dass das neue TKG von Dez 2021 gelte nicht für sie, das ist aktuell meine Baustelle. Keine Lust auszubauen, deshalb ne miese Internetanbindung liefern und trotzdem denn vollen Preis verlangen ist nicht

**Reply by qdlmcfresh** [reply]
![grafik](https://user-images.githubusercontent.com/10837173/160464979-9625c1fb-945b-49af-bb19-bd85936e0a38.png)
Breitband Desktop-App ist auch ein Witz ... Muss mir das mal automatisieren 😓 
(Die anderen LAN-Verbindungen sind zwar nur so sachen wie OpenVPN und VMWare aber die mach ich sicher nicht jedes mal alle einzeln aus um ne Messung durchzuführen :D )

**Comment by fda77** [comment]
Ich hatte diese "Desktop-App" einfach in eine VM gepackt...
